hail/python/hail/context.py

```diff
@@ -381,7 +394,55 @@ def tmp_dir():
     return Env.hc()._tmpdir
 
 
-def current_backend():
+def mktemp(*, suffix: str = '') -> DeletingFile:
+    """A temporary file that is deleted when it exits this context.
```

Comment:
"this context" is confusing, because the context (manager) is returned.  "it exists"  What's "it"?  How about "Returns a context manager which creates a temporary file name which is cleaned up when the context exits."?